infra/storage-account/application-storage-account.bicep

```diff
@@ -0,0 +1,88 @@
+@description('Required. The parameter object for the virtual network. The object must contain the name,skuName,resourceGroup and subnetPrivateEndpoints values.')
+param vnet object
+
+@description('Required. Name of your storage account. The parameter object for storageAccount.')
+param storageAccount object
+
+@description('Optional. Location for all resources.')
+param location string = resourceGroup().location
+
+@description('Required. Environment name.')
+param environment string
+
+@allowed([
+  'Storage'
+  'StorageV2'
+  'BlobStorage'
+  'FileStorage'
+  'BlockBlobStorage'
+])
+@description('Optional. Type of Storage Account to create for the storage account.')
+param kind string = 'StorageV2'
+
+@description('Optional. Sets the custom domain name assigned to the storage account. Name is the CNAME source.')
+param customDomainName string = ''
```

Comment:
The custom domain parameters are not being used, they should be removed. I think we need to keep the wrapper bicep modules with minimal parameters. if need arises in future, we can add more parameters.

Reply:
@kbabigumira I have implemented it now.